src/app/static/src/tests/setup.js

```diff
@@ -1,4 +1,7 @@
 import "@babel/polyfill";
+import $ from "jquery";
+
+global.$ = global.jQuery = $;
```

Comment:
tokenize2 contains a reference to the global jQuery object, so need this wired up for tests to work (in the browser we do have a global jQuery object so it runs fine.)